.github/workflows/Java.yml

```
@@ -40,6 +40,8 @@ jobs:
         uses: hendrikmuhs/ccache-action@main
```

Unrelated to this PR, but could we also

```suggestion
        uses: hendrikmuhs/ccache-action@v1.2.8
```

to avoid having to maintain our fork of the action?